### PR TITLE
Adding New profile to UKCore

### PR DESCRIPTION
**Pull request by KazeemHamzat**
Adding SD for FamilyMemberHistory to UKCore. I have maintained the status as a draft.

### CODE REVIEW

File: structuredefinitions/UKCore-FamilyMemberHistory.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  <abstract value="false" />
+  <type value="FamilyMemberHistory" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/FamilyMemberHistory" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="FamilyMemberHistory.instantiatesCanonical">
+      <path value="FamilyMemberHistory.instantiatesCanonical" />
+      <type>
+        <code value="canonical" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/PlanDefinition" />
```
Comment (omarisgreat) [inline structuredefinitions/UKCore-FamilyMemberHistory.xml:43]: You may need to also change the targets for identifier assigners to UKCore-Organization and note.author to UKCore resources if these are included in the Profile (you can speak to @johngeo to clarify if this is needed).

**Comment by omarisgreat** [comment]
Hi @KazeemHamzat, Could you also add the extensions for recorder and encounter (from the CareConnect FamilyMemberHistory) to your UKCore PR as we probably need to uplift those as well (they don't need to be added to the FMH profile yet as this is still in draft).

**Review by omarisgreat — CHANGES_REQUESTED** [review]

File: structuredefinitions/UKCore-FamilyMemberHistory.xml (hunk 1 above)
Comment (johngeo) [reply structuredefinitions/UKCore-FamilyMemberHistory.xml:43]: @KazeemHamzat and @omarisgreat, yes if the UK Core profile exists then it's recommended for those target references to point to the relevant UK Core profile. Hope that helps.

Comment (KazeemHamzat) [reply structuredefinitions/UKCore-FamilyMemberHistory.xml:43]: With exception of the UKCore-Questionnaire, all other SD referenced by instantiatesCanonical are not in the UK Core.
The remaining changes raised have been actioned. Further review is after Jan 3, 2023, as scheduled by @johngeo

**Comment by johngeo** [comment]
@KazeemHamzat and @omarisgreat  I would change the description to align to the format used in ProfileUKCore-Patient:  "This profile defines the UK constraints and extensions on the International FHIR resource [Patient](https://hl7.org/fhir/R4/Patient.html)".  Probably worth comparing this to the changes shown here: <https://github.com/NHSDigital/FHIR-R4-UKCORE-STAGING-MAIN/commit/1b9947933a1a724e104b96852e36346cab0431c2#diff-1e07337bb6eacb5c1744cafa9231c7afa72cb3bbd6ae69dfe9e651c8094b1c1a>, that illustrates the change to the description better.  The publication date, should changed, currently this states <date value="2022-12-15" /> Is the UKCore-FamilyMemberHistory is meant to be released with the 1.0.0 STU1 release of the UK CORE then it probably should be <date value="2022-12-16" /> Hope this helps.

**Comment by johngeo** [comment]
Hi @KazeemHamzat the description looks fine now.

**Review by johngeo — APPROVED** [review]
Looks fine.

**Pull request merged**